### PR TITLE
Release milestone 6.3.0

### PR DESCRIPTION
**Pull request by huayunh**
Releases React component library v6.3.0

#### Changes proposed in this Pull Request:

- Bug fixes
- New contents for dev doc (getting started, component catalog, themes)
- Narrower drawer item
- Dark theme

#### To Test:

- Just verify that everything exists

### CODE REVIEW

**Review by surajeaton — APPROVED** [review]

**Comment by joebochill** [comment]
Please plan to demo these fixes before merging and deploying this release

File: .github/workflows/laberler.yml
Comment (joebochill) [inline .github/workflows/laberler.yml:1]: There is a type in this filename...is this why we are having issues with the labeling?

File: docs/public/index.html [hunk 1]
```diff
@@ -1,65 +1,77 @@
...
-        <div id="root"></div>
-        <!--
+    <title>Brightlayer UI React Docs</title>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=%gaID%">
+    </script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { dataLayer.push(arguments); }
+        gtag('js', new Date());
+        gtag('config', '%gaID%', { send_page_view: false })
```
[the remark this answers is not in the document]
Comment (huayunh) [reply docs/public/index.html:60]: Let me have a look at this

[the remark this answers is not in the document]
Comment (huayunh) [reply docs/public/index.html:60]: Our inclination is "it aint break, dont fix", as the ga id is somehow still showing up properly in the deployed dev branch prod version

[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/public/index.html:60]: After looking at this again, we have 2 entries in the head tag. The first one seems to be correct but the second one not sure if that should be there. Its doing nothing and not sure why we would need it. Google Analytics seem to be ok and the second entry I'm unsure of

image
image

[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/public/index.html:60]: Going to try this like @joebochill mentioned

image

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**